README.md

```diff
@@ -14,29 +16,19 @@
 
 [Learn more](https://module-federation.io/)
 
-## 🔥 What does universe offer?
-
-The universe repository provides a set of tools to enhance module federation's capabilities. These include:
+## ✨ What new features does Module Federation 2.0 provide?
 
-- [Module Federation Runtime](https://module-federation.io/guide/basic/runtime.html)
-- [Rspack Module Federation](https://module-federation.io/guide/basic/rspack.html)
-- [Webpack Module Federation](https://module-federation.io/guide/basic/webpack.html)
-- [Next.js Module Federation](https://module-federation.io/guide/framework/nextjs.html)
+* 🎨 Module Federation Runtime
+* 🧩 Runtime Plugins System
+* 📝 Manifest
+* 🚀 Dynamic type prompt
+* 🛠️ Chrome Devtool
 
 ## 📚 Getting Started
 
 To get started with Module Federation, see the [Quick Start](https://module-federation.io/guide/start/quick-start.html).
 
 
-## 📦 Packages
```

Comment:
Why are these removed? These are community contributions and the official doc site. I don't know why they would be just dropped.

Reply:
Let me see where this would be better, I want to add this to the document

Reply:
We should have some location where packages are easy to observe as a whole, or we bounce them into our docs site if we have corresponding docs for that package else link to the code in the package. 

If we had "past generation" section otherwise we could just mark them in readme somewhere for user to find

Reply:
https://deploy-preview-2359--module-federation-docs.netlify.app/guide/start/npm-packages.html

I've added a chapter on npm packages just to list the packages and I thought that would make for a better presentation
![image](https://github.com/module-federation/core/assets/27547179/21868ca5-9572-4884-bd12-01138b786c23)